docpipeline/generate.py

```diff
@@ -168,29 +163,33 @@ def process_blob(blob, credentials, devsite_template, xrefs):
     # Rename the output TOC file to be _toc.yaml to match the expected
     # format. As well, support both toc.html and toc.yaml
     try:
-        shutil.move(
-            site_api_path.joinpath("toc.yaml"), site_api_path.joinpath("_toc.yaml")
-        )
+        shutil.move(site_path.joinpath("toc.yaml"), site_path.joinpath("_toc.yaml"))
     except FileNotFoundError:
-        shutil.move(
-            site_api_path.joinpath("toc.html"), site_api_path.joinpath("_toc.yaml")
-        )
-
-    # Copy the xrefmap file to the output directory.
-    xrefmap = site_path.joinpath("xrefmap.yml")
-    shutil.copy(xrefmap, site_api_path)
+        shutil.move(site_path.joinpath("toc.html"), site_path.joinpath("_toc.yaml"))
 
     # Add the prettyprint class to code snippets
-    add_prettyprint(site_api_path)
+    add_prettyprint(site_path)
```

Comment:
I suspect we want to delete manifest.json as well. That's docfx output that I doubt we want to publish (and won't have been publishing before)

Reply:
Note: if with the PR as it is, we *are* uploading manifest.json, but our golden tests aren't failing with unexpected data, we should fix the tests to spot that.

Reply:
Done. I added a test.

We don't have full on golden tests in this repo -- only doc-templates.